<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelSetupDialog</class>
 <widget class="QDialog" name="ChannelSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Channel setup</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListView" name="channelList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>178</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <widget class="QPushButton" name="removeItem">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addItem">
      <property name="text">
       <string>Add new</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>341</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="6" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Scale factor ([mV|nA]/V)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDoubleSpinBox" name="conversionFactor">
      <property name="minimum">
       <double>-100.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="reference"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Reference</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QDoubleSpinBox" name="offset">
      <property name="minimum">
       <double>-100.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Offset (mV|nA)</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QComboBox" name="readOffsetSource">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If set, the offset is read from the selected channel at the start of an experiment (and when reading/resetting in this dialog), overwriting the manually set value.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Channel type</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Comedi device</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="device"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Channel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Channel name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="name"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Read offset from:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="type">
      <item>
       <property name="text">
        <string>Analog IN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Analog OUT</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="channel"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Range</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="range"/>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="read_reset">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QDoubleSpinBox" name="read_reset_val">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Input channels: Displays the value read when the read button is clicked.&lt;/p&gt;&lt;p&gt;Output channels: Clicking reset applies this value to the channel. To zero the output completely, set the offset to 0 and the offset source to None before resetting.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="minimum">
       <double>-10000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChannelSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChannelSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addItem</sender>
   <signal>clicked()</signal>
   <receiver>ChannelSetupDialog</receiver>
   <slot>addChannel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeItem</sender>
   <signal>clicked()</signal>
   <receiver>ChannelSetupDialog</receiver>
   <slot>removeChannel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addChannel()</slot>
  <slot>removeChannel()</slot>
 </slots>
</ui>
